dll/win32/newdev/lang/bg-BG.rc

```diff
@@ -93,4 +93,7 @@ BEGIN
     IDS_CHSOURCE_TITLE "Choose search and install options"
     IDS_SEARCHDRV_TITLE "Searching..."
     IDS_INSTALLDRV_TITLE "Installing..."
+
+    /* For IDD_BROWSE_FOR_FOLDER */
```

Comment:
remove that comment and the empty line above for all languages. The title is absolutely self-description-like.
If everyone did it like that, the rc files would look rather ugly.
Same for all other languages!